osgtest/tests/test_550_condorce.py

```diff
@@ -126,9 +129,9 @@ def test_07_ceview(self):
         core.skip_ok_unless_installed('htcondor-ce-view')
         view_url = 'http://%s:%s' % (core.get_hostname(), int(core.config['condor-ce.view-port']))
         try:
-            src = urllib2.urlopen(view_url).read()
-        except urllib2.URLError:
-            self.fail('Could not reach HTCondor-CE View at %s' % view_url)
+            src = to_str(urlopen(view_url).read())
+        except EnvironmentError as err:
+            self.fail('Could not reach HTCondor-CE View at %s: %s' % (view_url, err))
```

Comment:
Huh, this shows up as `EnvironmentError` now?

Reply:
In Python 2 it's `urllib2.URLError` and in Python 3 it's `urllib.error.URLError`. Both are subclasses of EnvironmentError (which is also the parent of IOError and OSError, so it's a good catch-all for ... basically errors caused by something outside of your program). I could add URLError to the imports if you want me to be more specific.

Reply:
Nope it's fine, I was just curious more than anything!

Reply:
I divide exceptions into two categories: "my fault" and "not my fault" :-)